dmoj/checkers/bridged.py

```diff
@@ -10,23 +10,26 @@
 executor = None
 
 
-def get_executor(files, lang, compiler_time_limit, problem_id):
+def get_executor(files, lang, flags, cached, compiler_time_limit, problem_id):
     global executor
 
     if executor is None:
-        if not isinstance(files, list):
-            files = [files]
-        filenames = [os.path.join(get_problem_root(problem_id), f) for f in files]
-        executor = compile_with_auxiliary_files(filenames, lang, compiler_time_limit)
+        if isinstance(files, str):
+            filenames = [files]
+        elif isinstance(files.unwrap(), list):
+            filenames = list(files.unwrap())
```

Comment:
Wait why

Reply:
Same behaviour as generators

Reply:
Unless I'm missing something this is just `filenames = files`?

Reply:
No `filenames` is a `ConfigNode`